packages/artifacts/CurrentCost.json

```diff
@@ -0,0 +1,185 @@
+        "type": "constructor"
+      },
+      {
+        "anonymous": false,
+        "inputs": [
+          {
+            "indexed": true,
+            "name": "previousOwner",
+            "type": "address"
+          },
+          {
+            "indexed": true,
+            "name": "newOwner",
+            "type": "address"
+          }
+        ],
+        "name": "OwnershipTransferred",
+        "type": "event"
+      },
+      {
+        "constant": true,
+        "inputs": [
+          {
+            "name": "oracleAddress",
+            "type": "address"
+          },
+          {
+            "name": "endpoint",
+            "type": "bytes32"
+          },
+          {
+            "name": "start",
+            "type": "uint256"
+          }
+        ],
+        "name": "_currentCostOfDot",
+        "outputs": [
+          {
+            "name": "cost",
+            "type": "uint256"
+          }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+      },
+      {
+        "constant": true,
+        "inputs": [
+          {
+            "name": "oracleAddress",
+            "type": "address"
+          },
+          {
+            "name": "endpoint",
+            "type": "bytes32"
+          },
+          {
+            "name": "start",
+            "type": "uint256"
+          },
+          {
+            "name": "nDots",
+            "type": "uint256"
+          }
+        ],
+        "name": "_costOfNDots",
+        "outputs": [
+          {
+            "name": "cost",
+            "type": "uint256"
+          }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+      },
+      {
+        "constant": true,
+        "inputs": [
+          {
+            "name": "oracleAddress",
+            "type": "address"
+          },
+          {
+            "name": "endpoint",
+            "type": "bytes32"
+          }
+        ],
+        "name": "_dotLimit",
+        "outputs": [
+          {
+            "name": "",
+            "type": "uint256"
+          }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+      }
+    ],
+    "networks": {
+      "1": {
+        "events": {},
+        "links": {},
+        "address": "0xde775430f4e9f0df7887d6c7c3ce63b257300fca",
+        "transactionHash": "0x194246e0852f5bc02ec8d7659330ad7b6350ebef2876538e64769d8f2df7b8cb"
+      },
+      "42": {
+        "events": {},
+        "links": {},
+        "address": "0xbee49077ae340b4d418e334cf33fdf640db76f2f",
+        "transactionHash": "0xad2e13da45bfba1d2932310483b755a37f4c077a7dce6043df01763716190722"
+      },
+      "4447": {
+        "events": {},
+        "links": {},
+        "address": "0x8e4c131b37383e431b9cd0635d3cf9f3f628edae",
+        "transactionHash": "0x39e1009fa32a6419c5bdb35f33e3a7e48871a9eb6d5684de58b6637a05646291"
+      },
+      "development": {
```

Comment:
remember to add hardhat - see the pull request on monorepo

Reply:
Sorry for the delay. On it, thank you!